app/services/conviction_decision_tree.rb

```diff
@@ -47,7 +47,7 @@ def destination
   def after_conviction_subtype
     return edit(:conviction_bail)   if conviction_subtype.bailable_offense? && !step_name.eql?(:bypass_bail_conviction_subtype)
     return edit(:compensation_paid) if conviction_subtype.compensation?
-    return after_adult_motoring     if conviction_subtype.parent.inquiry.adult_motoring?
+    return after_adult_motoring     if conviction_subtype.parent.inquiry.adult_motoring? || conviction_subtype.parent.inquiry.youth_motoring?
```

Comment:
Here you can replace the two conditions with the method from the decorator `#conviction_subtype.motoring?`

Apart from that, this decision tree is coupled in a few places to adult motoring, as you can see by the naming `return after_adult_motoring` so it needs a bit of refactoring to make either generic so it works for both kind of motoring, or make sure where there are differences between youth and adult, these are correct.

Cukes are failing because of this.